sdk/keyvault/azure-security-keyvault-keys/pom.xml

```diff
@@ -41,6 +41,11 @@
         <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-surefire-plugin;external_dependency} -->
         <configuration>
           <rerunFailingTestsCount>3</rerunFailingTestsCount>
+          <argLine>
+            --add-exports com.azure.core/com.azure.core.implementation.http=ALL-UNNAMED
+            --add-opens com.azure.security.keyvault.keys/com.azure.security.keyvault.keys.cryptography=ALL-UNNAMED
+            --add-opens com.azure.security.keyvault.keys/com.azure.security.keyvault.keys=ALL-UNNAMED
+          </argLine>
```

Comment:
Since all other `--add-opens` are in `pom.client.xml`, it's better to move this there to keep them all in one place.

Reply:
OK. Will move there.